### DISCUSSION TITLE
Additional Prometheus metrics

### DISCUSSION THREAD

**Discussion by gabrieltz** [meilisearch/product#669] [Feedback & Feature Proposal]
Hello,

It would be nice to have some additional prometheus metrics that already exist in `/stats` endpoint:
1. `lastUpdate` exposed as a gauge with seconds since last update, or unix timestamp  to detect if something is not "feeding meilisearch with new data" so we can define alert rules to prevent stale data.
2. `isIndexing` exposed as 0/1 would be helpful to correlate slower http_response_time_seconds_bucket with indexing

and from the `/tasks` endpoint
1. exposing the number of tasks with queued status ( gauge )
2. exposing the number of tasks with processing status ( gauge ) 

thank you

**Comment by macraig** [comment]
Hi @gabrieltz , thanks for taking the time to leave your feedback! I'll move it to https://github.com/meilisearch/product/discussions/625 and close this so we can keep all the Prometheus metrics under the same discussion. Feel free to add to that discussion as well!

**Reply by gabrieltz** [reply]
thank you @macraig , i'll keep an eye on the discussion there
